app/src/main/java/org/mozilla/focus/home/PinSiteManager.kt

```diff
@@ -0,0 +1,232 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.home
```

Comment:
put new classes in rocket package

Reply:
Moved to org.mozilla.rocket.home.pinsite